src/Task.js

```diff
@@ -57,7 +57,6 @@ class Task {
 
             taskInfo.parameters.forEach((param) => {
               parameters[param.name] = param;
-              delete parameters[param.name].name;
```

Comment:
I don't see why this was removed, but that's probably because I don't fully understand the structure.  However, shouldn't we delete it still?  It's always a good idea to clean up after JS as much as possible